client/src/api/pastes.js

```diff
@@ -34,6 +36,16 @@ export function postPaste(paste) {
   }).then(response => response.json());
 }
 
+export function postPasteToSlack(slackPost) {
```

Comment:
the naming `postPasteToSlack` tells me that the argument should be a `paste`. but it's a slack post.